### ISSUE TITLE
Update detailed schedule in instructor notes

### ISSUE THREAD

**Issue by bast** [coderefinery/documentation#266]
This was the schedule from March 2023:

* 10:45 - 12:30 [Documentation](https://coderefinery.github.io/documentation/)
    - 10:45 - 10:55 [Motivation and tools](https://coderefinery.github.io/documentation/wishlist/)
        - create a wishlist in HackMD
    - 10:55 - 11:05 Break
    - 11:00 - 11:20 [Writing good README files](https://coderefinery.github.io/documentation/writing-readme-files/#writing-good-readme-files)
        - brief discussion
        - [15 minutes for exercises](https://coderefinery.github.io/documentation/writing-readme-files/#exercises), choosing either 1, 2 or 3 (or do multi
ple of time allows)
        - brief discussion
    - 11:20 - 12:00 [Sphinx and markdown](https://coderefinery.github.io/documentation/sphinx/)
        - discussion (5 min)
        - [Exercise 1 as type along (10 min)](https://coderefinery.github.io/documentation/sphinx/#exercise-sphinx-basics)
        - [Exercise 2, exercise 3 if time allows (20 min)](https://coderefinery.github.io/documentation/sphinx/#exercise-sphinx-content)
        - Discussion, going over exercise (5 min)
    - 12:00 - 12:10 Break
    - 12:10 - 12:25 [Deploying Sphinx documentation to GitHub Pages](https://coderefinery.github.io/documentation/gh_workflow/#deploying-sphinx-documentat
ion-to-github-pages)
        - [Exercise 1 as demo](https://coderefinery.github.io/documentation/gh_workflow/#exercise-deploy-sphinx-documentation-to-github-pages)
    - 12:25 - 12:30 [Summary](https://coderefinery.github.io/documentation/summary/)

**Comment by samumantha** [comment]
Suggestion for adaption to 2 hours for September workshop:

    10:00 - 10:05 Motivation and tools
        - create a wishlist in HackMD
    10:05 - 10:10 Writing good README files - brief discussion
    10:10 - 10:35 Exercises, choosing either 1, 2 or 3 (or do multiple of time allows)
    10:35 - 10:40  - brief discussion
    10:40 - 10:45 Sphinx and markdown- discussion 
    10:45 - 10:55 Exercise 1 as type along 
    10:55 - 11:05 Break
    11:05 - 11:35 Exercise 2, exercise 3 if time allows (20 min)
    11:35 - 11:40 Discussion, going over exercise (5 min)
    11:40 - 11:50 [Deploying Sphinx documentation to GitHub Pages](https://coderefinery.github.io/documentation/gh_workflow/#deploying-sphinx-documentat
    ion-to-github-pages)
        Exercise 1 as demo
    11:50 - 12:00 Summary

What do you think, @bast ? 
Basically more time to exercises (and 5 more min to demo in end)
